### PR TITLE
Firefox: Pass ScreenshotTests

### LINKED ISSUE #116: Pass screenshot.spec.js tests in Firefox

### PR DESCRIPTION
**Pull request by Meir017**
closes #116

### CODE REVIEW

File: src/PlaywrightSharp.Tests/Page/ScreenshotTests.cs [hunk 1]
```diff
@@ -250,13 +251,13 @@ public async Task ShouldAllowTransparency()
 
         ///<playwright-file>screenshot.spec.js</playwright-file>
         ///<playwright-describe>Page.screenshot</playwright-describe>
-        ///<playwright-it>should allow transparency</playwright-it>
+        ///<playwright-it>should render white background on jpeg file</playwright-it>
```
Comment (kblok) [inline src/PlaywrightSharp.Tests/Page/ScreenshotTests.cs:254]: <3

**Comment by Meir017** [comment]
mac & ubuntu are having issues in CI :(

**Comment by kblok** [comment]
It seems we have consistent fails on MacOS. I'll take a look at it.

**Comment by kblok** [comment]
@Meir017 fixed :)

**Pull request merged**